data/json/monsters/feral_humans.json

```diff
@@ -60,6 +60,26 @@
       "DROPS_AMMO"
     ]
   },
+  {
+    "id": "mon_chud",
+    "type": "MONSTER",
+    "name": { "str": "feral troglobite" },
+    "description": "One of the still-living crazed humans, changed by something they are exposed to down here.  Their face is dominated by unnaturally large, faintly glowing eyes and needle-sharp teeth poking through their lips.  They remind you of something out of a campy monster movie - and of the good times when underground cannibals were less of a real concern.",
+    "copy-from": "mon_feral_human_pipe",
+    "looks_like": "mon_zombie_runner",
+    "symbol": "@",
+    "color": "white",
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 4,
+    "melee_cut": 3,
+    "dodge": 4,
+    "vision_night": 15,
+    "special_attacks": [ { "id": "scratch", "min_mul": 0.5, "cooldown": 10 }, { "type": "bite", "min_mul": 0.7, "cooldown": 15 } ],
+    "death_drops": "default_zombie_death_drops",
+    "zombify_into": "mon_zombie_shady",
+    "extend": { "flags": [ "FILTHY" ] }
```

Comment:
Would extend work to add "max_dist": 10 (from the original) to the path settings?

Reply:
I omitted `max_dist` on purpose - in my opinion if a monster shouldn't become aggroed at range X then they either need lower vision range or appropriate anger triggers, not get their pathfinding handicapped (especially since we tell the player "this can see you" and "this wants to attack you", but *not* "this would attack you if you got a single tile closer").

Reply:
max_dist 10 is a reduced pathfinding? What's the default?

Reply:
....well *****. Okay, so after digging into it, actually most monsters don`t pathfind at all beyond beelining for the player in a straight line. That's a nice little info tidbit not to mention in the monster documentation, but it explains a lot. The monster pathfinding for "some select monsters" got added 5 years ago, then stayed completely undocumented (that's a running theme, sadly) until one year ago. 

Thanks for bringing that up, I guess I'll go look for supposedly-smart mobs that don't show it.

Reply:
The same is true of when they're fleeing, BTW... something that, at least for ones that have a max_path above 0, really needs some fixing. Fleeing uses a goal point of the "reflected" position of the player... even if that reflected position is so far away the monster can't pathfind to it. Adjusting the goal point to a closer one, still in the same direction, that _was_ within the max_path distance would be kind of nice. (Doing something like this for pathfinding beyond the max_path in general - as in, if it's too far away, try for a point partway there - would be good also; not sure if anything of that sort happens currently, but I don't think so.)